### PR TITLE
Updated Redirects.py 

### LINKED ISSUE #7983: Redirect `/privacy/products/` to `/firefox/privacy/products/`
## Description

We should [add a redirect](https://bedrock.readthedocs.io/en/latest/redirects.html) to point **/privacy/products/** to **[/firefox/privacy/products/](https://www.mozilla.org/firefox/privacy/products/)**.

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #7983

### CODE REVIEW

**Comment by Patil2099** [comment]
@alexgibson  Is this correct ?

**Comment by alexgibson** [comment]
@Patil2099 circleCI is reporting some failures above. Please take a look and see if you can work out what needs changing. If you get stuck, let me know!

File: bedrock/privacy/redirects.py [hunk 1]
```diff
@@ -20,4 +20,8 @@
 
     # mozilla/bedrock/#5745 - archive cliqz policy
     redirect(r'^privacy/firefox-cliqz/?$', 'privacy.archive.firefox-cliqz-2018-06'),
+
+    # mozilla/bedrock/#7983
+    redirect(r'^/privacy/products/','firefox.privacy.products'),
```
Comment (alexgibson) [inline bedrock/privacy/redirects.py]:
```suggestion
    redirect(r'^/privacy/products/', 'firefox.privacy.products'),
```

File: tests/redirects/map_globalconf.py [hunk 2]
```diff
@@ -1284,4 +1284,7 @@
 
     # Issue 7970
     url_test('/moss/mission-partners-india/', '/moss/mission-partners/'),
+
+    # Issue 7983
+    url_test('/privacy/products/', 'firefox.privacy.products'),
```
Comment (alexgibson) [inline tests/redirects/map_globalconf.py]:
```suggestion
    url_test('/privacy/products/', 'firefox/privacy/products'),
```

File: bedrock/privacy/redirects.py [hunk 3]
```diff
@@ -20,4 +20,8 @@
 
     # mozilla/bedrock/#5745 - archive cliqz policy
     redirect(r'^privacy/firefox-cliqz/?$', 'privacy.archive.firefox-cliqz-2018-06'),
+
+    # mozilla/bedrock/#7983
+    redirect(r'^/privacy/products/', 'firefox.privacy.products'),
```
Comment (alexgibson) [inline bedrock/privacy/redirects.py]: This redirect fails when the trailing slash is omitted.
```suggestion
    redirect(r'^/privacy/products/?$', 'firefox.privacy.products'),
```

**Pull request merged**